src/content/adhoc/accessible-names/guidance.tsx

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { create, GuidanceTitle, React } from '../../common';
+
+export const guidance = create(({ Markup, Link }) => (
+    <React.Fragment>
+        <GuidanceTitle name={'Accessible names'} />
+
+        <h2>Why Accessible names matter</h2>
+
+        <p>
+            The Accessible name is a short label that provides information about the purpose of a user interface element. Without it, people
+            that use Assistive Technologies (AT) would have difficulty understanding the purpose of an element and interacting with it.
+        </p>
+        <p>
+            For more information about Accessible names and their computation, please check out <Link.accessileNameComputation />.
+        </p>
+
+        <h2>About Accessible names visualization</h2>
+
+        <p>
+            The visualization in Accessibility Insights for Web enable developers to see accessibility markup that’s normally invisible. The{' '}
+            <Markup.Term>Accessible names</Markup.Term> visualization highlights elements that have accessible names and, when possible,
+            shows the accessible name. It does not highlight elements with empty, or no accessible names.
+        </p>
+
+        <Markup.Columns>
+            <Markup.Do>
+                <h3>
+                    If a widget has visible label, make sure they are programmatically related to it.{' '}
+                    <Link.WCAG21UnderstandingInfoAndRelationships /> <Link.WCAG21UnderstandingLabelInName />
```

Comment:
You can use the existing links for WCAG guidelines here. Also missing some punction from the guidance doc: `(<Link.WCAG_1_3_1 />, <Link.WCAG_2_5_3 />)`